vtm/src/org/oscim/backend/DateTime.java

```diff
@@ -37,4 +37,10 @@ public int getSecond() {
     public int getDayOfYear() {
         return Calendar.getInstance().get(Calendar.DAY_OF_YEAR);
     }
+
+    @Override
+    public int getTimeZoneOffset() {
+        Calendar inst = Calendar.getInstance();
+        return inst.getTimeZone().getOffset(inst.getTimeInMillis()) / 60000;
```

Comment:
Better use the found milliseconds on supported platforms. And let apps decide on precision?

Reply:
Ok your choice :wink: